.zenodo.json

```diff
@@ -0,0 +1,99 @@
+{
+    "title": "CLDF dataset derived from Chacon's \"A revised proposal of Proto-Tukanoan consonants and Tukanoan family classification\" from 2014",
+    "access_right": "open",
+    "keywords": [
+        "cldf:Wordlist",
+        "linguistics"
+    ],
+    "creators": [
+        {
+            "name": "Thiago Costa Chacon"
+        },
+        {
+            "name": "Tiago Tresoldi"
+        },
+        {
+            "name": "Sacha Beniamine"
+        }
+    ],
+    "contributors": [
+        {
+            "name": "Thiago Costa Chacon",
+            "type": "Editor"
+        },
+        {
+            "name": "Johann-Mattis List",
+            "type": "DataCurator"
+        }
+    ],
+    "communities": [
```

Comment:
Which lexibank version do you use? I know we have this bug, by which the "communities" will be expanded each time, rather than reduced.

Reply:
But I thought we had a solution already.

Reply:
it is pylexibank==2.7.0:
https://github.com/lexibank/chacontukanoan/blob/a6e41d4fc5efde65172b9b4008c794303eee47b2/cldf/requirements.txt#L30

Reply:
Fix is already in pylexibank but we haven't done a release since then.